test/Tester/GrainActivatorTests.cs

```diff
@@ -25,27 +22,19 @@ public class Fixture : BaseTestClusterFixture
         {
             protected override TestCluster CreateTestCluster()
             {
-                var options = new TestClusterOptions(1);
-                options.UseSiloBuilderFactory<TestSiloBuilderFactory>();
-                return new TestCluster(options);
+                var builder = new TestClusterBuilder(1);
+                builder.AddSiloBuilderConfigurator<TestSiloBuilderFactory>();
+                return builder.Build();
             }
 
-            private class TestSiloBuilderFactory : ISiloBuilderFactory
+            private class TestSiloBuilderFactory : ISiloBuilderConfigurator
```

Comment:
nit: you could rename to `TestSiloBuilderConfigurator`

Reply:
ah, yeah, I renamed some, but I'm sure I missed many